client/lib/components/learn_page_promo.dart

```diff
@@ -6,6 +8,17 @@ import 'package:who_app/components/promo_curved_background.dart';
 import 'package:who_app/components/themed_text.dart';
 import 'package:who_app/constants.dart';
 
+const blurRadius = 20.0;
+const shadow = Shadow(blurRadius: blurRadius);
```

Comment:
I'd remove the `const blurRadius = 20.0;` and just inline it since the value isn't used anywhere else.
```suggestion
const shadow = Shadow(blurRadius: 20.0);
```